src/pages/guides/tutorials/grids-addon.md

```diff
@@ -0,0 +1,1321 @@
+  const redHex = hex.slice(0, 2);
+  const greenHex = hex.slice(2, 4);
+  const blueHex = hex.slice(4, 6);
+  // Convert hex values to decimal values
+  const red = parseInt(redHex, 16) / 255;
+  const green = parseInt(greenHex, 16) / 255;
+  const blue = parseInt(blueHex, 16) / 255;
+  return utils.createColor(red, green, blue);
+};
+```
+
+It'd be nice to group rows and columns. The Editor class provides a [`createGroup()`](/references/scriptruntime/editor/classes/Editor.md#creategroup) method returning a [`GroupNode`](/references/scriptruntime/editor/classes/GroupNode/). Like all `ContainerNode` classes, it has a `children` property, which we can append rectangles to.
+
+```js
+const addRows = (rowsNumber, gutter, color) => {
+  // ...
+  var rows = [];
+  // ...
+
+  const rowsGroup = editor.createGroup();          // creating a group
+  page.artboards.first.children.append(rowsGroup); // appending to the page
+  rowsGroup.children.append(...rows);              // appending rectangles
+};
+// 👆 same in addColumns()
+```
+
+![](images/grids-addon-groups.png)
+
+<!-- code below -->
+<InlineAlert variant="warning" slots="heading, text1, text2" />
+
+**Grouping elements**
+
+Like every other node, groups are first created, and then put on the page. It's important to remember that **groups can only act as containers if they exist on the document** first.
+
+```js
+// DO
+const rowsGroup = editor.createGroup();          // ✅  create
+page.artboards.first.children.append(rowsGroup); // ✅  append
+rowsGroup.children.append(...rows);              // ✅  fill
+
+// DON'T
+const rowsGroup = editor.createGroup();          // ✅  create
+rowsGroup.children.append(...rows);              // ❌  fill 
+page.artboards.first.children.append(rowsGroup); // ❌  append
+```
+
+To complete the projects, we can add some finishing touches. Groups can be locked: preventing accidental shifts and transformations would be nice indeed. The Reference documentation comes in handy again with the boolean [`locked`](/references/scriptruntime/editor/classes/GroupNode.md#locked) property, which we can easily set after populating the group.
+
+```js
+// ...
+rowsGroup.children.append(...rows);
+rowsGroup.locked = true;
+```
+
+The Reference also shows an interesting [`blendMode`](/references/scriptruntime/editor/classes/GroupNode.md#blendmode): setting it to [`multiply`](/references/scriptruntime/editor/enums/BlendModeValue/#multiply) will produce a visually nicer overlay effect ([opacity](/references/scriptruntime/editor/classes/GroupNode.md#opacity) can be an alternative).
+
+```js
+// ...
+rowsGroup.blendMode = Constants.BlendModeValue.multiply;
+rowsGroup.locked = true;
+```
+
+At present, the Reference includes only a few enumerations, such as `BlendModeValue`. As the Document API expands, more enumerations will be added. They provide sets of named constants, making the code more readable by replacing direct numeric values with descriptive names.[^4]
```

Comment:
Would be good to reinforce to the dev that they should avoid hard-coding the values of the enums in their code. While I _hope_ these values stay pretty constant, there's no guarantee, and so devs should always rely on using the enum instead.

Reply:
There's a footnote, but I should transform it into a warning box for better visibility.